datasets/flwr_datasets/utils.py

```diff
@@ -85,3 +86,135 @@ def _check_if_dataset_tested(dataset: str) -> None:
             f"The currently tested dataset are {tested_datasets}. Given: {dataset}.",
             stacklevel=1,
         )
+
+
+def divide_dataset(
+    dataset: Dataset, division: Union[List[float], Tuple[float, ...], Dict[str, float]]
+) -> Union[List[Dataset], DatasetDict]:
+    """Divide the dataset according to the `division`.
+
+    The division support varying number of splits, which you can name. The splits are
+    created from the beginning of the dataset.
+
+    Parameters
+    ----------
+    dataset: Dataset
+        Dataset to be divided.
+    division: Union[List[float], Tuple[float, ...], Dict[str, float]]
+        Configuration specifying how the dataset is divided. Each fraction has to be
+        >0 and <1. They have to sum up to at most 1 (smaller sum is possible).
+
```

Comment:
Worth showing as inline examples how to construct divisions from both list of floats (probably the most common thing people'd use) and dicts ?

Reply:
Done